### ISSUE TITLE
[combobox]: axe reports a11y violation for invalid attr value

### ISSUE THREAD

**Issue by williamernest** [uber/baseweb#4726]
Axe tools reports an a11y violation on the combobox component during the initial page load since this element is "aria-controls" by an ID value that does not yet exist in the DOM (the popover is not showing, so it doesn't get rendered. In this instance, the controls attr should be left off. This may cause a different a11y issue, so we should explore the appropriate approach here and document it. 
![image](https://user-images.githubusercontent.com/34519388/151222667-3dd019c4-eca8-4918-9986-4b155a9f7087.png)
